### PR TITLE
Fix loader bug in tagcloud context screen

### PR DESCRIPTION
**Pull request by anishsarangi**
NO loader was showing up when the API call was taking some time. I have fixed that

### CODE REVIEW

File: scripts/tagcloud.js [hunk 1]
```diff
@@ -20,12 +20,13 @@ function get_tags (url) {
   var not_display3 = not_display4 + ' extension'
   var dontarray = ['view page', 'open', 'read more', not_display1, not_display2, not_display3, not_display4]
   var new_url = 'https://archive.org/services/context/tagcloud?url=' + toBeUsedAsURL
-  $('#loader_tagcloud').show()
+  $('#container-wordcloud').hide()
   fetch(new_url)
     .then(response => response.json())
     .then(function (data) {
+      $('#container-wordcloud').show()
```
Comment (cgorringe) [inline scripts/tagcloud.js:27]: the .show() on line 27 should be after the if on line 29.

Comment (anishsarangi) [reply scripts/tagcloud.js:27]: YES, correct. I am changing it sorry !1

**Comment by anishsarangi** [comment]
I have made changes @cgorringe . Please review !

**Comment by cgorringe** [comment]
Goes **after** the **if**:
`if (!data.error && data.length > 0) {`

**Comment by anishsarangi** [comment]
@cgorringe Please check the else condition. I have already handled that case.

**Comment by cgorringe** [comment]
Looks good.

**Pull request merged**